Source/WebKit/_WebKit_SwiftUI/Foundation+Extras.swift

```diff
@@ -35,16 +35,55 @@ func onNextMainRunLoop(do body: @escaping @MainActor () -> Void) {
 }
 
 struct AnyEquatable: Equatable {
-    let value: Any
-    private let equals: (Any) -> Bool
+    private enum Storage {
+        case value(Any, (Any) -> Bool)
+        case never
+    }
+
+    private let storage: Storage
+
+    var value: Any {
```

Comment:
Should `value` also be `private`? Maybe call it `base` to match [`AnyHashable`’s impl](https://github.com/swiftlang/swift/blob/1576ba4448631225363f8397ff635378b78e1cd8/stdlib/public/core/AnyHashable.swift#L189)?